data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,184 @@ mission "Coalition: Outskirts Gauger Ready"
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		random < 30
+		credits > 140
+		has "Coalition: Contributor: done"
+	on offer
+		conversation
+			`Upon entering the spaceport, you walk past a bookstore, currently holding a book signing event for an author known as 'Ann Oryza Sativa'. Nobody pays attention to her. It appears as if she is a struggling author.`
+			choice
+				`	(Ignore her.)`
+					decline
+				`	(Buy one of her books.)`
+					goto "buy her book"
+				`	(Sit down and talk to her.)`
+			`	You sit down on the table, next to Ann. She greets you with a smile. "Hello! Would you like to buy one of my books?" she asks.`
+			choice
+				`	"What are your books about?"`
+				`	"No, I'm not interested. How has life been as an author though?"`
+					goto "life as an author sucks"
+			label "what are her books about"
+			`	"I mostly write science fiction books. Today, I'm here to promote 'Beyond the Hyperdrive', a story in which a group of humans steal a bunch of jump drives from the Quarg. If I continue past this point I'd be spoiling it. Read it for yourself, you'll like it!"`
+			choice
+				`	"I'm interested. Wait here, I'll go buy myself a copy."`
+					goto "buy her book"
+				`	"Why is it a story about the Quarg? Do you hate the Quarg or something?"`
+					goto "do you hate the quarg"
+				`	"Cool. How has life been lately as an author?"`
+			label "life as an author sucks"
+			`	"Bad," says Ann. "I've never had a bestseller or even a book that surpasses 10,000 copies sold."`
+			choice
+				`	"What are your books about?"`
+					goto "what are her books about"
+				`	"Well, that's gotta hurt. I hope you find success in your life one day.`
+					decline
+			label "buy her book"
+			action
+				set "beyond the hyperdrive book"
+				payment -60
+			`	You enter the bookstore, ask the employee for one of Ann's books, and purchase it for 60 credits.`
+			choice
+				`	(Now go to Ann and get it signed.)`
+				`	(Leave.)`
+					decline
+			`	You move to the table where Ann is sitting. She looks visibly happy as she grabs your book, takes out a pen, and signs it. "Thanks for buying one of my books. I don't get much sales and people like you mean the world to me."`
+			choice
+				`	"I appreciate it. How has life as an author been though?"`
+					goto "life as an author sucks"
+				`	"Thanks for signing my copy. I hope you find success in your life. Goodbye."`
+					decline
+			label "do you hate the quarg"
+			`	"I have a lot of problems with them. You see, there are so many systems we can't reach and the Quarg have the secrets to reaching them. And yet they won't give humanity the secret. They are trying to restrict us. Keeping us all locked in cages!"`
+			`	"So, I wrote 'Beyond the Hyperdrive' to share my feelings about the Quarg. Humanity will one day succeed! We will surpass them and make humanity even greater beyond its wildest dreams!"`
```

Comment:
```suggestion
			`	"I have a lot of problems with them. You see, there are so many systems we can't reach and the Quarg have the secrets to reaching them. And yet they won't give humanity the secret. They are trying to restrict us. Keeping us all locked in cages!`
			`	"So, I wrote 'Beyond the Hyperdrive' to share my feelings about the Quarg. Humanity will one day succeed! We will surpass them and make humanity even greater beyond its wildest dreams!"`
```
I can't find an example to show, but when there are 2 consecutive paragraphs of the same character talking, and there's no break in their dialog between the paragraphs, the second one doesn't end with a quotation, to kinda show it's still going or something of the sort.
Alternatively can just join the 2 paragraphs, if you prefer.